### DISCUSSION TITLE
原创文章 - Sui 节点质押模型

### DISCUSSION THREAD

**Discussion by ShirleneLiu** [starcoinorg/web3startrek#117] [Call for Article]
Move 生态技术文章

任务描述：撰写原创文章+直播讲解文章

题目：Sui 节点质押模型

要求示例：
1.质押怎么操作
2.质押经济模型
3.质押代码分析

时间截止：2023.05.15

任务奖励：70U +  MoveFuns DAO 100积分

首发：DAOrayaki 策展

任务跟进： @geometryolife @uvd

**Comment by gofenix** [comment]
之前没接触过这个，想提前看看有英文文档或者源码路径没，我去阅读学习一下。然后尝试写一篇学习文章投稿
